src/ledctl.c

```diff
@@ -806,13 +814,13 @@ status_t slot_execute(struct slot_request *slot_req)
 		return list_slots(slot_req);
 	case OPT_SET_SLOT:
 		status = slot_req->get_slot_fn(slot_req->device, slot_req->slot, &slot_res);
+		if (status != STATUS_SUCCESS)
```

Comment:
Isn't it make the code below dead?

Reply:
This is what the code was before

https://github.com/intel/ledmon/blob/3ac01d97bba08f47b2432c9cc44dd1e76366dbba/src/ledctl.c#L807-L818

Without moving the status check up, we are doing a comparison with bad/unknown `slot_req->state` in the set slot path.  I believe we should ensure that the `get_stot_fn` was successful before moving forward with additional logic?